pkgs/tools/package-management/nix/default.nix

```diff
@@ -39,7 +41,8 @@ common =
 
       nativeBuildInputs =
         [ pkgconfig ]
-        ++ lib.optionals is24 [ autoreconfHook autoconf-archive bison flex libxml2 libxslt docbook5 docbook_xsl_ns jq ];
+        ++ lib.optionals is24 [ autoreconfHook autoconf-archive bison flex libxml2 libxslt
+                                docbook5 docbook_xsl_ns jq gmock git mercurial ];
```

Comment:
Don't add git and mercurial, we already have way too many dependencies.

Reply:
Done.